internal/entities/info.go

```diff
@@ -30,3 +31,9 @@ type Server struct {
 	Interface string
 	Uptime    time.Duration
 }
+
+type Map struct {
+	Center         GeoJSONLocation
+	BoundSouthWest GeoJSONLocation
+	BoundNorthEast GeoJSONLocation
```

Comment:
These attributes are not GeoJSON, GeoJSON is different. This should also be an array as well

GeoJson definition: https://datatracker.ietf.org/doc/html/rfc7946

Reply:
Ahh okay, you are using the latitude and longitude fields of this struct, I think a simple array will do the job here.